easybuild/easyconfigs/q/Qt6/Qt6-6.5.2-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,94 @@
+easyblock = 'CMakeNinja'
+
+name = 'Qt6'
+version = '6.5.2'
+
+homepage = 'https://qt.io/'
+description = "Qt is a comprehensive cross-platform C++ application framework."
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+# disabling use of -ftree-vectorize is required to avoid compilation failures on some systems (e.g. Intel Skylake X)
+toolchainopts = {'vectorize': False}
+
+source_urls = [
+    'https://download.qt.io/official_releases/qt/%(version_major_minor)s/%(version)s/single/',
+    'https://download.qt.io/archive/qt/%(version_major_minor)s/%(version)s/single/',
+    'https://download.qt.io/new_archive/qt/%(version_major_minor)s/%(version)s/single/',
+]
+sources = ['qt-everywhere-src-%(version)s.tar.xz']
+patches = [
+    # they may not be needed if qmake is not used?
+    # QT_CFLAGS_ARCH_HASWELL in cmake/QtCompilerOptimization.cmake should be patched?
+    # 'Qt5-5.13.1_fix-avx2.patch',
+    # 'Qt5-5.13.1_fix-qmake-libdir.patch',
+    # 'Qt5-5.15.2_fix-qtwebengine-breakpad-glibc-2.34.patch',
```

Comment:
Looks these patches are not needed anymore. I may remove them in the next commit.